actions/setup-sigstore-env/run-containers.sh

```diff
@@ -15,7 +15,7 @@
 # limitations under the License.
 
 echo "setting up OIDC provider"
-pushd ./fakeoidc
+pushd ./fakeoidc || exit
```

Comment:
`exit` has the same issue as "set -e" in a script that is sourced...

Reply:
ok. `|| return` also works and won't exit the shell.